DUI3-DX/Converters/ArcGIS/Speckle.Converters.ArcGIS3/Utils/IFeatureClassUtils.cs

```diff
@@ -2,17 +2,20 @@
 using Objects.GIS;
 using Speckle.Converters.Common.Objects;
 using Speckle.Core.Models;
+using FieldDescription = ArcGIS.Core.Data.DDL.FieldDescription;
 
 namespace Speckle.Converters.ArcGIS3.Utils;
 
 public interface IFeatureClassUtils
 {
   void AddFeaturesToFeatureClass(
     FeatureClass newFeatureClass,
-    VectorLayer target,
-    List<string> fieldAdded,
-    IRawConversion<Base, ACG.Geometry> gisGeometryConverter
+    List<GisFeature> gisFeatures,
+    List<FieldDescription> fields,
+    IRawConversion<IReadOnlyList<Base>, ACG.Geometry> gisGeometryConverter
   );
-
-  ACG.GeometryType GetLayerGeometryType(VectorLayer target);
+  public object? FieldValueToNativeType(FieldType fieldType, object? value);
+  public List<FieldDescription> GetFieldsFromSpeckleLayer(VectorLayer target);
+  public FieldType GetFieldTypeFromInt(int fieldType);
+  public ACG.GeometryType GetLayerGeometryType(VectorLayer target);
```

Comment:
This seems weird. A utility class should not be performing conversions as well, or it becomes a converter in it's own right.

This is kinda hiding some conversion logic but I'm not fully grasping it's use yet, so I'm hesitant to propose moving it anywhere yet.

Reply:
this function is writing the features into the table(layer), and indeed, calling the geometry converters from within